intel_extension_for_transformers/neural_chat/examples/serving/README.md

```diff
@@ -0,0 +1,62 @@
+# Serving NeuralChat Text Generation with Triton Inference Server
```

Comment:
Shall we enable chatbot by leveraging the triton inference server?

Reply:
Yes, we exactly are using triton inference server to deploy a chatbot.